docs/monitors/expvar.md

```diff
@@ -0,0 +1,206 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# expvar
+
+This monitor exports metrics derived from [expvar](https://golang.org/pkg/expvar/)
+variables in JSON objects retrieved from a HTTP endpoint.
+
+The monitor uses configured JSON paths to get metric and dimension values from the fetched JSON objects.
+[Memstats metrics](https://github.com/signalfx/integrations/blob/master/expvar/docs/expvar_metrics.md)
+are created by default. They do not require any configuration. They are derived from variable
+[memstats](https://golang.org/pkg/runtime/) that is exposed by default.
+
+Below is a sample YAML configuration showing the minimal configuration of the expvar monitor. Only
+memstats metrics will be exported for this configuration. The `extraDimensions` field is not required but recommended.
+```
+monitors:
+- type: expvar
+  host: 172.17.0.3
+  path: /debug/vars
+  port: 8000
+```
+It is recommended to add the `metric_source` extra dimension as show below in order to facilitate filtering in
+SignalFx.
+```
+monitors:
+- type: expvar
+  host: 172.17.0.3
+  path: /debug/vars
+  port: 8000
+  extraDimensions:
+    metric_source: expvar
+```
+
+Below is a sample YAML configuration showing the configuration of custom metrics `xyz` and `h.i`. The `name`s
+of metrics and dimensionals are optional. The `JSONPath` is used for `name` instead if not provided.
+```
+monitors:
+- type: expvar
+  host: 172.12.0.6
+  path: /debug/vars
+  port: 8123
+    metrics:
+      - name: xyz
+        JSONPath: a.b.c.d
+        type: counter
+        dimensions:
+          - name: rst
+            JSONPath: a.b
+      - JSONPath: h.i
+        type: gauge
+      - JSONPath: j
+        type: gauge
+        dimensions:
+          - name: kl
+            value: mno
+  extraDimensions:
+    metric_source: expvar
+```
+
+Below is an example JSON object that the above configuration may apply to.
+```
+{
```

Comment:
Show a realistic expvar output with the standard fields instead of this highly abstract JSON document.